src/libvirtApi/common.js

```diff
@@ -418,15 +421,16 @@ function startEventMonitorStoragePools(connectionName) {
         });
 }
 
-function storagePoolStopOrUndefine(connectionName, poolPath) {
-    return call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "ListStoragePools", [0], { timeout, type: "u" })
-            .then(objPaths => {
-                if (objPaths[0].includes(poolPath))
-                    return storagePoolGet({ connectionName, id: poolPath, updateOnly: true });
-                else // Transient pool which got undefined when stopped
-                    return store.dispatch(undefineStoragePool({ connectionName, id: poolPath }));
-            })
-            .catch(ex => console.warn("storagePoolStopOrUndefine action failed:", ex.toString()));
+async function storagePoolStopOrUndefine(connectionName, poolPath) {
+    try {
+        const [objPaths] = await call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "ListStoragePools", [0], { timeout, type: "u" });
+        if (objPaths.includes(poolPath))
+            await storagePoolGet({ connectionName, id: poolPath, updateOnly: true });
+        else // Transient pool which got undefined when stopped
+            store.dispatch(undefineStoragePool({ connectionName, id: poolPath }));
+    } catch (ex) {
+        console.warn("storagePoolStopOrUndefine action failed:", ex.toString());
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)